### DISCUSSION TITLE
Read file by choosing it with html/PlutoUI

### DISCUSSION THREAD

**Discussion by ghost** [fonsp/Pluto.jl#791] [Questions about using Pluto]
Hallo,

is there any possibility making the file path interactive, for example with html "<input type='file'\>"  or PlutoUI, so the path is stored as a String? Maybe something similar to the "Open from File:" field at the start page.

**Comment by fonsp** [comment]
Unfortunately not: this interaction would need to happen in the web page, but the web page does not have access to your file system. If you pick a file, the browser hides all metadata except the contents and the filename.

If you know the list of possible files in advance (e.g. you can use `readdir()` to get all files in a folder), then you can use a `PlutoUI.Select`.
